test/sql/aggregate/aggregates/test_arg_min_max_null.test

```diff
@@ -0,0 +1,163 @@
+# name: test/sql/aggregate/aggregates/test_arg_min_max_null.test
+# description: Test argminnull and argmaxnull operator
+# group: [aggregates]
+
+statement ok
+PRAGMA enable_verification
+
+statement ok
+PRAGMA verify_external
+
+#Corner cases
+statement error
+select argminnull()
+----
+
+query I
+select argminnull(NULL,NULL)
+----
+NULL
+
+query I
+select argminnull(1,1)
+----
+1
+
+statement error
+select argminnull(*)
+----
+
+query I
+select argminnull(i,i) from range (100) tbl(i);
+----
+0
+
+query I
+select argminnull(i,i) from range (100) tbl(i) where 1 = 0;
+----
+NULL
+
+statement error
+select argmaxnull()
+----
+
+query I
+select argmaxnull(NULL,NULL)
+----
+NULL
+
+query I
+select argmaxnull(1,1)
+----
+1
+
+statement error
+select argmaxnull(*)
+----
+
+query I
+select argmaxnull(i,i) from range (100) tbl(i);
```

Comment:
I think it would be useful to test `FIRST(x ORDER BY y)` and `LAST(x ORDER BY y)` in the tests since that is how they will be used in practice (and it should test both the aggregate and the rewrite at the same time).

Reply:
We actually have a lot of tests that do this (I kept tripping over them...) `test/sql/aggregate/aggregates/test_first_last_any_ordered.test` comes to mind.